.gitignore

```diff
@@ -17,5 +17,6 @@
 /spec/examples.txt
 /tmp
 /public/uploads
+/lib/assets/data/soi_extract.csv
```

Comment:
This file has to be generated only once, for the initial import, and the name is not going to change at the moment, we can address eventual changes later on.